bedrock/etc/urls.py

```diff
@@ -2,5 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+from bedrock.mozorg.util import page
+
 urlpatterns = (
+    page('firefox/retention/thank-you-a', 'firefox/retention/thank-you-a.html'),
+    page('firefox/retention/thank-you-b', 'firefox/retention/thank-you-b.html')
```

Comment:
I know it's a little weird, but our convention on bedrock for template folders is `{app name}/templates/{app name}/whatever`. Can you move the HTML templates from `/templates/firefox/retention` into `/templates/etc/firefox/retention`?